pvlib/location.py

```diff
@@ -356,3 +359,92 @@ def get_sun_rise_set_transit(self, times, method='pyephem', **kwargs):
                              'one of pyephem, spa, geometric'
                              .format(method))
         return result
+
+
+def lookup_altitude(latitude, longitude, filepath=None):
+    """
+    Look up location altitude from local low-resolution altitude map
+    supplied with pvlib. The data for this map comes from multiple open data
+    sources with varying resolutions aggregated by Mapzen.
+
+    More details can be found here
+    https://github.com/tilezen/joerd/blob/master/docs/data-sources.md
+
+    Altitudes from this map are a coarse approximation and can have
+    significant errors (100+ meters) introduced by downsampling and
+    source data resolution.
+
+    Parameters
+    ----------
+    latitude : float.
+        Positive is north of the equator.
+        Use decimal degrees notation.
+
+    longitude : float.
+        Positive is east of the prime meridian.
+        Use decimal degrees notation.
+
+    filepath : None or string, default None
+        The path to the ``.h5`` file.
+
+    Returns
+    -------
+    altitude : int
```

Comment:
why return `int`? I would expect a float for a single value.

Reply:
Because of the current encoding of the altitude map, you will never get an altitude with a fractional component.
But I think you're right float seems more appropriate here.
That way, if we increase the resolution or do other changes, we wouldn't have to change the API.
I will change it to float